style.css

```diff
@@ -96,21 +96,44 @@ img {
   max-height: max-content;
 }
 
+img[src$=".svg"] {
```

Comment:
I just learned about this selector today. It lets you pick just the img elements whose src filename ends with `.svg`. You could use the same approach to apply a style to other image types as well (like `.png` or `.jpg`).

Reply:
Cool tip!